kubernetes/helm/openrag/values.yaml

```
@@ -233,6 +233,29 @@ backend:
+    loadDefaultFlows:
+      resources:
+        requests:
+          ephemeral-storage: "500Mi"  # Ephemeral storage for downloading default flows
+        limits:
+    waitForLangflow:
+      resources:
+        requests:
+          ephemeral-storage: "10Mi"  # Minimal storage for busybox network check loop
+        limits:
+          ephemeral-storage: "10Mi"
+          ephemeral-storage: "500Mi"
```

YAMLlint (1.38.0)

[error] duplication of key "ephemeral-storage" in mapping

(key-duplicates)